rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/LongRunningAction.java

```diff
@@ -106,6 +106,11 @@ public LongRunningAction(LRAService lraService, Uid rcvUid) {
         this.scheduler = Executors.newScheduledThreadPool(1);
     }
 
+    // used for MBean LRA listing, see com.arjuna.ats.arjuna.tools.osb.mbean.ObjStoreBrowser
+    public LongRunningAction(Uid rcvUid) {
+        this(null, rcvUid);
```

Comment:
The variable `lraService` is expected to be non null so better is to just construct one:

``` java
this(new LRAService(), rcvUid);
```

Reply:
+1, thank you